src/usr/local/www/bandwidth_by_ip.php

```diff
@@ -119,10 +181,10 @@
 	// echo $bandwidthinfo;
 	$emptyinfocounter = 1;
 	if ($bandwidthinfo != "") {
-		$infoarray = explode (":", $bandwidthinfo);
+		$infoarray = explode (";", $bandwidthinfo);
 		if (($filter == "all") ||
-		    (($filter == "local") && (ip_in_subnet($infoarray[0], $intsubnet))) ||
-		    (($filter == "remote") && (!ip_in_subnet($infoarray[0], $intsubnet)))) {
+			(($filter == "local") && ((ip_in_subnet($infoarray[0], $intsubnet)) || (ip_in_subnet($infoarray[0], $intsubnet6)))) ||
+			(($filter == "remote") && ((!ip_in_subnet($infoarray[0], $intsubnet)) || (!ip_in_subnet($infoarray[0], $intsubnet6))))) {
```

Comment:
Multiline statements should use 4 spaces to indent, as it was before the change. Please leave it as it is